content/tutorials/jwt-authentication.textile

```diff
@@ -136,96 +139,97 @@ This step shows you how to create the server code from scratch. If you get stuck
 
 ```[javascript]
-const jwtPayload = {                                                                                                                                                                                                                                           
-    'x-ably-capability': JSON.stringify({ '*': ['publish', 'subscribe'] }),                                                                                                                                                                                    
-    'x-ably-clientId': 'OPTIONAL_CLIENT_ID'                                                                                                                                                                                                                    
-}                                                                                                                                                                                                                                                              
-                                                                                                                                                                                                                                                               
-const jwtOptions = {                                                                                                                                                                                                                                           
-    expiresIn: ttlSeconds,                                                                                                                                                                                                                                     
-    keyid: `${keyName}`                                                                                                                                                                                                                                         
-}      
-```
+const express = require('express');
 
-9. To finish the web app you use "Express":https://www.npmjs.com/package/express to build a simple server to serve files on your localhost:
+const app = express();
 
-```[javascript]
-const express = require('express'),
-    app = express();
 app.use('/', express.static(__dirname))
 ```
 
 The Express server uses a middleware to serve the content, and uses @express.static(__dirname)@ to specify that any HTML files in the root folder are served directly.
 
-10. Create the @/auth@ route in this server. This route is responsible for creating a JWT and sending this token back to the client:
+8. Create the @/auth@ route in this server. This route is responsible for creating a JWT and sending this token back to the client. Normally we would validate the User before issuing the JWT token but for simplicity we'll just use their input as the `clientId`.
```

Comment:
User -> user

General note: Don't mix "you" and "we". Use "you" if it's something the user does, "we" if it's something Ably does. E.g. "At Ably, we provide ..."

Suggestion: "In a production application you would check that the user exists and is authorized to use your service before issuing a JWT with their user ID as the @clientId@. In this example you will just use the user ID they provide."